src/components/Footer/FooterLinks/FooterLinksItem.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import cx from 'classnames';
+
+type FooterLinksItemProps = {
+  className?: string;
+  text: string;
+  url: string;
+  target?: string;
+  rel?: string;
+  role?: string;
+  tabIndex?: number;
```

Comment:
order alphabetically (trying to find a way to include this in eslint, however haven't found a working one yet!)